client/src/components/dlg/new-listing.html

```diff
@@ -0,0 +1,34 @@
+<template>
+  <ux-dialog class='new-listing-dlg'>
```

Comment:
@Vheissu put together some modal styles that are pretty nice, can we switch these those styles?  It should be reflected in the other modal code (ex here - https://github.com/AureliaToolbox/gigme.io-client/blob/master/client/src/components/show-all-addresses.html)